data-migration/src/main/java/com/quorum/tessera/data/migration/H2DataExporter.java

```diff
@@ -1,21 +1,20 @@
 package com.quorum.tessera.data.migration;
 
 import java.nio.file.Path;
-import java.sql.Connection;
-import java.sql.DriverManager;
-import java.sql.PreparedStatement;
 import java.sql.SQLException;
-import java.sql.Statement;
 import java.util.Map;
-import java.util.Map.Entry;
 
 public class H2DataExporter implements DataExporter {
 
     private static final String INSERT_ROW = "INSERT INTO ENCRYPTED_TRANSACTION (HASH,ENCODED_PAYLOAD) VALUES (?,?)";
 
-    private static final String CREATE_TABLE = "CREATE TABLE ENCRYPTED_TRANSACTION "
-        + "(ENCODED_PAYLOAD LONGVARBINARY NOT NULL, "
-        + "HASH LONGVARBINARY NOT NULL UNIQUE, PRIMARY KEY (HASH))";
+    private static final String CREATE_TABLE = "CREATE TABLE " +
```

Comment:
Like you've done for the tests, could we not use the DDL scripts to do the creation? It would prevent future issues - maybe even create some `INSERT` DDL scripts, although this would be specific for the migration, so may not be appropriate.

Reply:
Also applies to other DB types, but won't duplicate the comment.

Reply:
Also, didn't we provide the ability for custom scripts for other database types? I seem to remember saying we could move the migration towards using that model, seems like an appropriate time to do that.

Reply:
the work to bundle and separate the create table lines is more hassle than the benefit . We'd need to remove the ddl module and perhaps integrate with the db migration, but then we'll need to redocument. The ddl module should never been added, but now it is we need to live with it for now.

Reply:
Change added in response to comments. Good catch. We'll need to manually test as the unit tests don't cover the reading and execution of statements from ddls.